algorithms/kernel/dtrees/forest/classification/df_classification_train_container.h

```diff
@@ -66,6 +66,8 @@ services::Status BatchContainer<algorithmFPType, method, cpu>::compute()
 
     decision_forest::classification::Model * m = result->get(classifier::training::model).get();
 
+    m->setNFeatures(x->getNumberOfColumns());
```

Comment:
Seems the same problem is for DF regression. Could you create issue on it or fix it also in this PR?

Reply:
This issue has already been created: https://github.com/intel/daal/issues/307.
I will create a new pr with a fix for DF regression.